app/controllers/admin/dashboard_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Admin::DashboardController < ApplicationController
+
+  before_action :require_admin
+
+  def require_admin
+    render file: "/public/404" unless current_admin?
+  end
+
+  def index
+  end
```

Comment:
To stick to convention, we could do
` def index; end`

Reply:
We're on it!